packages/shared/src/lib/dateFormat.ts

```diff
@@ -14,6 +14,7 @@ const oneWeek = 7 * oneDay;
 const oneMonth = 30 * oneDay;
 export const oneYear = oneDay * 365;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why not delete this if unused.

Reply:
probably so we can re-use it in the future, and not re-invent it?

Reply:
Makes sense. Although I'd probably delete this line personally. Love seeing red lines on PRs 😂

Reply:
Yeah, implementation looked valuable. I can export it and then I don't have to add the comment

Reply:
removed comment